packages/flutter/lib/src/widgets/media_query.dart

```diff
@@ -314,6 +314,20 @@ class MediaQueryData {
 
   /// Whether the device is inverting the colors of the platform.
   ///
+  /// {@tool snippet}
+  /// Setting it, for instance by using:
```

Comment:
This snippet is leaving me with more questions.... what are you trying to document with this?

Reply:
You are right, it is not clear. In my opinion, given that the documentation for the three properties `invertColors`, `highContrast`, and `boldText` is very similar, I expected them to behave the same.

You can override the platform's value when you set `boldText`.
You cannot override the platform's value when you set `invertColors` or `highContrast`.

I will change the code by use these words. But if it is unclear only to me then let me know, I will close the PR.